code/modules/research/techweb/_techweb.dm

```diff
@@ -298,6 +298,7 @@
 /datum/techweb/proc/complete_experiment(datum/experiment/completed_experiment)
 	available_experiments -= completed_experiment
 	completed_experiments[completed_experiment.type] = completed_experiment
+	log_research("[completed_experiment.name] ([completed_experiment.type]) has been completed on techweb id [id]")
```

Comment:
This functionality [already exists](https://github.com/tgstation/tgstation/blob/master/code/modules/admin/admin_investigate.dm); we don't need to create a new proc for it. 

`investigate_log("[key_name(user)] researched [id]([json_encode(price)]) on techweb id [stored_research.id].", INVESTIGATE_RESEARCH)`

Reply:
Investigate log is for atoms, we cant call it on datums.